portfolio/src/main/webapp/index.html

```diff
@@ -16,13 +16,25 @@ <h2>Click here for a random fact about me:</h2>
             <button onclick="addRandomGreeting()">Random Fact</button>
             <div id="greeting-container"></div>
       </div>
-        <!--row of photos -->
+        
+        <form action="data" method="POST">
+            <p>What is a fun fact about you? </p>
+            <textarea name="text-input"> type here... </textarea>
+            <br/></br>
```

Comment:
Instead of using like break elements, try using css styling to add some margin below the textarea element or above the input element.